spec/models/in_person_enrollment_spec.rb

```diff
@@ -88,6 +88,60 @@
     end
   end
 
+  describe 'email_reminders' do
```

Comment:
These tests look good, but maybe add some more enrollments that have the `early_reminder_sent` and/or `late_reminder_sent` flags set to `true` (and corresponding values for other relevant fields) to test that aspect of the filters.